### PR TITLE
Using set instead of append to add auth header.

### PR DESCRIPTION
**Pull request by sanbornsen**

### CODE REVIEW

**Comment by sanbornsen** [comment]
This is fix of issue - https://github.com/almighty/almighty-ui/issues/191

@nimishamukherjee @joshuawilson @michaelkleinhenz @aslakknutsen @SMahil please review it.

File: src/app/shared/broadcaster.service.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+export class Broadcaster {
+  private _eventBus: Subject<BroadcastEvent>;
+
+  constructor() {
+    this._eventBus = new Subject<BroadcastEvent>();
+  }
+
+  broadcast(key: any, data?: any) {
+    console.log("I am here");
```
Comment (nimishamukherjee) [inline src/app/shared/broadcaster.service.ts]: Remove this console.log :)

**Comment by nimishamukherjee** [comment]
I'm fine with the changes.

**Comment by michaelkleinhenz** [comment]
Looks good. General suggestion: I think we have to do more source comments. That would help to understand what the source is actually supposed to do.

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**